app/components/Calendar.client.tsx

```diff
@@ -0,0 +1,11 @@
+import TUICalendar from '@toast-ui/react-calendar';
```

Comment:
```suggestion
import ToastUICalendar from '@toast-ui/react-calendar';
```

Makes it more readable